ctapipe_io_magic/__init__.py

```diff
@@ -182,6 +210,9 @@ def __init__(self, input_url=None, config=None, parent=None, **kwargs):
 
         self._subarray_info = self.prepare_subarray_info()
 
+        if not self.is_mc:
```

Comment:
`EventSource` must already implement `is_simulation`

Reply:
Right, I changed to `self.is_simulation` where possible.